Resources/Storage/StorageElement.py

```diff
@@ -9,6 +9,7 @@
 import errno
 import threading
 import sys
+from functools import reduce
```

Comment:
This is a buggy import that keeps coming but I don't know why. probably someone has autopep8 configured for python3

Reply:
In this PR it was simply moved (it was few lines below), is it incorrect?

Reply:
Well, not incorrect per se, but rather useless now :-) 
https://docs.python.org/2/library/functools.html#functools.reduce

Reply:
I see. I leave it here anyway.